packages/ir-sdk/fern/apis/ir-types-latest/generators.yml

```diff
@@ -18,7 +18,7 @@ groups:
         output:
           location: npm
           url: npm.buildwithfern.com
-          package-name: '@fern-fern/ir-sdk'
+          package-name: '@fern-fern/ir-v34-sdk'
```

Comment:
lets keep this `ir-sdk` otherwise will break a shit ton of imports in typescript bc you import from the package directly

Reply:
Updated!